flink-runtime/src/test/java/org/apache/flink/runtime/rest/RestEndpointITCase.java

```diff
@@ -237,8 +300,8 @@ public TestParameters getUnresolvedMessageParameters() {
 	}
 
 	private static class TestParameters extends MessageParameters {
-		private final JobIDPathParameter jobIDPathParameter = new JobIDPathParameter();
-		private final JobIDQueryParameter jobIDQueryParameter = new JobIDQueryParameter();
+		protected final JobIDPathParameter jobIDPathParameter = new JobIDPathParameter();
```

Comment:
what is this change for?

Reply:
To make it accessible when you have a `FaultyTestParameter` instance. Can also cast it which gives access to `jobIDPathParameter`.